StickyHeaders/src/main/java/io/doist/recyclerviewext/sticky_headers/StickyHeadersLinearLayoutManager.java

```diff
@@ -603,4 +698,43 @@ private void sortHeaderAtIndex(int index) {
             }
         }
     }
+
+    public static class SavedState implements Parcelable {
+        private Parcelable superState;
+        private int pendingScrollPosition;
+        private int pendingScrollOffset;
+
+        public SavedState() {
+        }
+
+        public SavedState(Parcel in) {
+            superState = in.readParcelable(StickyHeadersLinearLayoutManager.class.getClassLoader());
```

Comment:
Shouldn't this be `SavedState.class.getClassLoader()`? (Theoretically they are the same, but asking for correctness :smile:)